dist/image.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/buildpack/pack/style"
 )
 
+const (
+	MixinsLabel = "io.buildpacks.stack.mixins"
+)
```

Comment:
This constant doesn't feel like it belongs here alongside these generic label getter & setters. Everywhere else `Get` or `Set` `Label` is consumed the label name is sourced from somewhere else.

Reply:
Done.